Dockerfile

```diff
@@ -64,7 +64,9 @@ COPY --from=builder /usr/lib/libfdb_c.so /usr/lib/
 COPY --from=builder /usr/lib/fdb /usr/lib/fdb/
 COPY --chown=fdb:fdb --from=builder /var/log/fdb/.keep /var/log/fdb/.keep
 
-USER fdb
+# Specify the fdb user by UID to satisfy Kubernetes PSPs, which 
+# require numeric UIDs to be assured a pod is not running as root
```

Comment:
```suggestion
# require numeric UIDs to be assured a container is not running as root
```

Reply:
This PR may be superceded by #425, which implements the same fix, but also adds securityContext to the pod/container :)

Reply:
If so please close this PR